### DISCUSSION TITLE
Propert way to Unload Screen?

### DISCUSSION THREAD

**Discussion by 1080papaya** [mathoudebine/turing-smart-screen-python#98] [Q&A]
Hi im still new with coding, i was able to follow the guides on how to create my own themes.
I can see there's a ctrl+c interrupt handler (signum) built in, but this only works for me before gpu information is loaded (cannot type in cmd once gpu information is being processed to the screen).

I've been physically disabling the usb from the turing smart screen as a work around.
Is there a way i can create a .py to unload/reset the screen or am I doing it wrong completely?

**Comment by mathoudebine** [comment]
Hi @1080papaya
You're right that the signal handler does not seem to work for Windows platform.
This may be linked to the GPUtil library that get the GPU information.
Can you open a bug here for this? https://github.com/mathoudebine/turing-smart-screen-python/issues/new/choose

In the meantime, if you launch the program from a command-line, closing the command window should stop the program (the screen should not update anymore after) but it is not a clean stop because of the signal handler not working...
When you restart it, the program will reset the screen automatically before running again. But sometimes it can crash and needs to be relaunched a second time to work, because of the same issue.

What is your use case for the program? Do you start it manually?

**Reply by 1080papaya** [reply]
I'll post this to the open bug section.

Bare with me here since im still a newbie.
as for my use case im starting main.py through command line (py main.py)

I ended up creating a file called kill.py and a batch file to first kill python in my processes and then load kill.py -  which is basically identical to display.py except i've removed as much code as i can so that it only executes         self.lcd.Reset().

The screen does need to be physically unplugged sometimes though as i've encounted random artifcating after several resetting although i may have a differnt use case scenerio than most other people.

I'm using Playnite (emulator/game launcher) to automatically load a .bat file based on the game i launch, the batch file will load up the correseponding game theme file that i have created (modified theme with a background based on the game im playing).
upon quitting the game playnite will execute my batch file to unload the screen.
